superset-frontend/plugins/plugin-chart-echarts/src/components/Echart.tsx

```diff
@@ -61,6 +63,15 @@ import {
 import { LabelLayout } from 'echarts/features';
 import { EchartsHandler, EchartsProps, EchartsStylesProps } from '../types';
 
+// Define this interface here to avoid creating a dependency back to superset-frontend,
+// and to appease the compiler.
+// ref: superset-frontend/src/explore/types
+interface ExplorePageState {
+  common: {
+    locale: string;
+  };
+}
```

Comment:
Oof.. I agree with the workaround here, but could we rephrase the comment to a `// TODO:` to move the type to `@superset-ui/core`?